sa/model.go

```diff
@@ -361,6 +361,23 @@ func modelToChallenge(cm *challModel) (core.Challenge, error) {
 	return c, nil
 }
 
+type recordedSerialModel struct {
+	ID             int64
+	Serial         string    `db:"serial"`
```

Comment:
Style nit: we've mostly ignored adding these `db` tags as they are only really necessary when the column name doesn't match the field name (ignoring case).